test/object_store/test_basic.py

```diff
@@ -270,3 +270,26 @@ async def test_object_storage_config(manager: ManagerClient, tmpdir, s3_server):
     print('Scylla assumes the only endpoint specified is the default one')
     MinioServer.create_conf_file(s3_server.address, s3_server.port, 'bad_key', 'bad_secret', 'region', obj_cfg)
     server = await manager.server_add(config=cfg)
+
+@pytest.mark.asyncio
+async def test_get_object_store_endpoints(manager: ManagerClient, tmpdir, s3_server):
+    obj_cfg = tmpdir / 'object_storage.yaml'
+    cfg = {'object_storage_config_file': str(obj_cfg)}
+
+    print('Scylla returns the object storage endpoints')
+    MinioServer.create_conf_file(s3_server.address, s3_server.port, 'bad_key', 'bad_secret', 'bad_region', obj_cfg)
+    MinioServer.append_endpoint_to_conf('localhost', s3_server.port, 'worse_region', obj_cfg, default=True)
+    server = await manager.server_add(config=cfg)
+
+    resp = await manager.api.get_object_store_endpoints(server.ip_addr)
+    endpoints = sorted(resp, key=lambda d: d['region'])
```

Comment:
It's fragile that the comparison below depends on some specific "order" of entries. If you want to make sure that the list returned by the API matches some other list, convert both to `set`s and compare sets, they are not sensitive to elements order

Reply:
It's 2 element lists and the list on the right operand is hardcoded. I removed the sort on the 1 element list below which is silly, my bad.